src/main/java/nextstep/subway/line/domain/entity/LineSections.java

```diff
@@ -106,7 +106,7 @@ private LineSection createNewDownSection(LineSection newLineSection,
             existingSection.getLine(),
             newLineSection.getDownStation(),
             existingSection.getDownStation(),
-            existingSection.getDistance() - newLineSection.getDistance()
+            existingSection.getDistance().getDistance() - newLineSection.getDistance().getDistance()
```

Comment:
두 객체의 내부값을 직접 꺼내어 연산하기보다는 요청과 응답방식으로 바꿔보는건 어떨까요. 
```suggestion
            existingSection.getDiffDistance(newLineSection.getDistance());
```